### DISCUSSION TITLE
https not showing as secure

### DISCUSSION THREAD

**Discussion by Gizmo1739** [quasarframework/quasar#14490] [General - Components / Directives / etc]
I'm building a site that tries to embed clips from twitch but the site needs to be HTTPS

So I tried to use the dev server setting

`https: true`

but it still does not show as secure

then i tried to use the mkcert to make a local SSL Certification then my code looked like this 
```
    https: {
      key: 'Full\\Path\\toKey\\DaGizApp\\ssl\\cert.key',
      cert: 'Full\\Path\\toCert\\DaGizApp\\ssl\\cert.crt',
    },
```
And still no dice 

This is my first time having to create my own cert but I've been doing trial and error research for a couple hours now 

any help would be appreciated

**Comment by richardmckenna** [comment]
Did you find any solution to this? Having the same issue with a fresh project.
